circus-train-gcp/pom.xml

```diff
@@ -27,39 +46,40 @@
     <dependency>
       <groupId>com.google.cloud.bigdataoss</groupId>
       <artifactId>gcs-connector</artifactId>
-      <version>1.6.1-hadoop2</version>
+      <version>${google-cloud-bigdataoss.version}-hadoop2</version>
```

Comment:
Aah, OK, I had forgotten that we already have all these GCP dependenices here, I thought it was only in the CT-GCP sub-module. Hmmm. We should maybe have a wider discussion where this should live but I guess this is consistent with what we've been doing up to now. Where does this "google-cloud-bigdataoss.version" property come from?